flake.nix

```diff
@@ -1,22 +1,21 @@
 {
   description = "Nix expressions for defining Replit development environments";
-  inputs.nixpkgs.url = "github:nixos/nixpkgs?rev=52e3e80afff4b16ccb7c52e9f0f5220552f03d04";
-  inputs.nixpkgs-unstable.url = "github:nixos/nixpkgs?rev=87f9156865ab09e3bde39aadb4131ae364ae704e";
-  inputs.prybar.url = "github:replit/prybar?rev=65f486534054665f1b333689417c39acd370d3a5";
+  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-23.05";
+  inputs.nixpkgs-unstable.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+  inputs.prybar.url = "github:replit/prybar";
 
   outputs = { self, nixpkgs, nixpkgs-unstable, prybar, ... }:
     let
-      mkPkgs = system: import nixpkgs {
+      mkPkgs = nixpkgs-spec: system: import nixpkgs-spec {
         inherit system;
         overlays = [ self.overlays.default prybar.overlays.default ]; # ++ import ;
+        # replbox has an unfree license
+        config.allowUnfree = true;
```

Comment:
maybe we change the license? or just import directly from the [repo](https://github.com/replit/replbox) idk why it was pulled in